docs/github-actions-ci/index.md

```diff
@@ -0,0 +1,156 @@
+# CI pipeline for a Spring Boot application using Github actions
+
+## Introduction
+What is Continous integration? "Continuous Integration is a software development practice where members of a team integrate their work frequently, usually each person integrates at least daily - leading to multiple integrations per day." [1]
```

Comment:
**[misspell]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>"Continous" is a misspelling of "Continuous"